bootloaders/riotboot/Makefile

```diff
@@ -14,6 +14,7 @@ CFLAGS += -DRIOTBOOT
 
 # Disable unused modules
 CFLAGS += -DNDEBUG -DLOG_LEVEL=LOG_NONE
+DISABLE_MODULE += core_init core_msg
```

Comment:
What about `core_panic`, shouldn't it be disabled as well?

Reply:
> What about `core_panic`, shouldn't it be disabled as well?

Yes, that'll be another PR.

Reply:
> Yes, that'll be another PR.

actually, that doesn't make sense. They both require exactly the same testing.